services/contrib/PrometheusScrapeAgent/config

```diff
@@ -0,0 +1,6 @@
+{
+    cache_timeout: 660,
+    tag_delimiter_re: "\s+|:|_|\.|/"
```

Comment:
Can you add documentation to what this is actually for?  Our preprocessor strips out # comments so explain the tag delimiter argument.